tree/tree/test/BulkApiMultiple.cxx

```diff
@@ -58,22 +296,18 @@ TEST_F(BulkApiMultipleTest, stdRead)
    TTreeReaderValue<float> myF(myReader, "myFloat");
    TTreeReaderValue<double> myG(myReader, "myDouble");
    Long64_t idx = 0;
-   float idx_f = 1;
-   double idx_g = 2;
+   float idx_f = fFloatInitial;
+   double idx_g = fDoubleInitial;
    Int_t events = fEventCount;
    sw.Start();
    while (myReader.Next()) {
       if (R__unlikely(idx == events)) {break;}
-      idx_f++;
-      idx_g++;
-      if (R__unlikely((idx < 16000000) && (*myF != idx_f))) {
-         printf("Incorrect value on myFloat branch: %f, expected %f (event %lld)\n", *myF, idx_f, idx);
-         ASSERT_TRUE(false);
-      }
-      if (R__unlikely((idx < 15000000) && (*myG != idx_g))) {
-         printf("Incorrect value on myDouble branch: %f, expected %f (event %lld)\n", *myG, idx_g, idx);
-         ASSERT_TRUE(false);
-      }
+      ASSERT_TRUE(compare(*myF, idx_f))
```

Comment:
These should really all be `EXPECT_TRUE`: `ASSERT` ends the program right here if it fails, makes sense for `nullptr` checks of pointers used later, not so for these consistency checks where it's interesting to see whether a change causes tons of comparisons to fail or just one.

Reply:
The original code had the assert.  The 'interesting' question is whether the ton of message is useful information.   Here, I expect that if the first comparison fails, every single ones will fail (i.e. 10e7 messages).
Note that it seems that the `ASSERT_*` only quit the test and not the program.